pet006/sub-01/pet/sub-01_pet.json

```diff
@@ -0,0 +1,36 @@
+{
+	"AcquisitionMode": "list mode",
+	"AttenuationCorrection": "measured(emission present), 0.096000 cm-1, attenuation smooth",
+	"AttenuationCorrectionMethod": "measured(emission present), 0.096000 cm-1, attenuation smooth",
+	"BodyPart": "Phantom",
+	"DecayCorrectionFactor": [1.42614],
+	"DoseCalibrationFactor": 1323,
+	"FrameDuration": [98000],
+	"FrameTimesStart": [0],
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"InjectedMass": 181149988.3691438,
+	"InjectedMassUnits": "ug",
+	"InjectedRadioactivity": 75.84999999999999,
+	"InjectedRadioactivityUnits": "MBq",
+	"InjectedVolume": 0,
+	"InjectionStart": 0,
+	"InstitutionName": "NIH Clinical Center, USA",
+	"Manufacturer": "GE MEDICAL SYSTEMS",
+	"ManufacturersModelName": "GE Advance",
+	"ModeOfAdministration": "infusion",
+	"ReconFilterType": "None",
```

Comment:
Schema is checking for  `none`    and not `None`

Reply:
```suggestion
	"ReconFilterType": "none",
```